src/service/club.ts

```diff
@@ -4,7 +4,12 @@ import * as axios from 'axios'
 // Client
 import ClubHubClient from '../client'
 import Club from 'src/models/club'
+import Section from 'src/models/section'
 
+export interface PublicClubInfo {
```

Comment:
This has the same name as another interface in this PR, might want to make the name more specific on one of them?